backend/controllers/prisonerProfile/prisonerQuickLook.js

```diff
@@ -158,7 +163,7 @@ module.exports = ({ prisonerProfileService, prisonApi, telemetry }) => async (re
               sentenceData.sentenceDetail &&
               sentenceData.sentenceDetail.releaseDate &&
               moment(sentenceData.sentenceDetail.releaseDate).format('D MMMM YYYY')) ||
-            extractLifeImprisonmentStatus(prisonerDataResponse, prisoner, unableToShowDetailMessage),
+            (await extractLifeImprisonmentStatus()),
```

Comment:
This function name is a bit misleading - actually it gets the Imp Status label

Reply:
Agreed - renamed